data/ecosystem/integrations.yaml

```diff
@@ -201,3 +201,13 @@
   docsUrl: https://github.com/fluentci-io/fluentci-engine#-opentelemetry-tracing
   components: [Rust]
   oss: true
+- name: Azure SDKs
+  url: https://github.com/Azure/azure-sdk-for-net
+  docsUrl: https://azure.github.io/azure-sdk-for-net/
```

Comment:
this should ideally point to the part of the documentation where it speaks about OpenTelemetry